app/services/datastore/application_rehydration.rb

```diff
@@ -79,5 +90,11 @@ def outgoings
 
       Outgoings.new(parent.outgoings.serializable_hash)
     end
+
+    def not_means_tested?
+      return true if means_tested.value == :no
+
+      false
```

Comment:
For historic applications `means_tested` will be `nil` but as they have all been passported applications, none would be 'not means tested' - so this needs to be set to false if means_tested is nil - it will be an always asked, mandatory question moving forward